packages/components/src/utils/validation/throwIfComponentIsDeprecated.ts

```diff
@@ -0,0 +1,5 @@
+import { getTagName } from '../../utils';
+
+export const throwIfComponentIsDeprecated = (host: HTMLElement, message: string): void => {
```

Comment:
That's not throwing anything 🙈

Reply:
What would you suggest as function name? `warnIfCOmponentIsDeprecated`?